src/pathplanning.org

```diff
@@ -90,7 +90,7 @@
   - Recurse!
 
 * STOx's Planner
-http://imgur.com/Ea040em
+[[file:http:/i.imgur.com/Ea040em.png]]
```

Comment:
This actually should be `http://` not `http:/`. It looks fine in the slides, but probably will break docs/gh previews, so I'll fix this if you don't get around to it in the next commit.